evals/cli/oaieval.py

```diff
@@ -121,17 +121,20 @@ class OaiEvalArguments(argparse.Namespace):
     dry_run_logging: bool
 
 
-def send_slack_message(result: dict[str, Any]) -> None:
+def send_slack_message(result: dict[str, Any], question) -> None:
```

Comment:
it is not always a question, right?
```suggestion
def send_slack_message(result: dict[str, Any], prompt) -> None:
```